modules/gapi/include/opencv2/gapi/infer/ie.hpp

```diff
@@ -117,75 +140,180 @@ template<typename Net> class Params {
               , 1u} {
     };
 
-    Params<Net>& cfgInputLayers(const typename PortCfg<Net>::In &ll) {
+    /** @brief Specifies sequence of CNN input layers names for inference.
```

Comment:
I wouldn't hardcode to `CNN` networks, it works with any networks right ?

Reply:
Changed. network instead CNN.